libr/anal/esil.c

```diff
@@ -14,6 +14,11 @@
 		R_BIT_UNSET (&esil->flags, FLG (x));\
 	}
 
+typedef union {
```

Comment:
its ok to use unions in internal apis, not public ones, otherwise interfacing with other programming languages is a pain